cypress/integration/Menu/position/index.js

```diff
@@ -3,33 +3,36 @@ import { Given, Then, When } from 'cypress-cucumber-preprocessor/steps'
 Given(
     'there is enough space to the right of the MenuItem to fit the SubMenu',
     () => {
-        cy.visitStory('MenuItem', 'Default position')
+        cy.visitStory('Menu', 'Default Position')
     }
 )
 
 Given(
     'there is not enough space to the right of the MenuItem to fit the SubMenu',
     () => {
-        cy.visitStory('MenuItem', 'Flipped position')
+        cy.visitStory('Menu', 'Flipped Position')
     }
 )
 
 Given(
     'there is not enough space to the right or the left of the MenuItem to fit the SubMenu',
     () => {
-        cy.visitStory('MenuItem', 'Shift into view')
+        cy.visitStory('Menu', 'Shift Into View')
     }
 )
 
-When('the user hovers over the MenuItem', () => {
-    cy.get('[data-test="dhis2-uicore-menuitem"]').trigger('mouseover')
+When('the user clicks on the MenuItem', () => {
+    cy.get('[data-test="dhis2-uicore-menuitem"]').click()
 })
 
 Then(
     'the right of the MenuItem is aligned with the left of the SubMenu',
     () => {
         getMenuItemAndSubMenuRects().then(([menuItemRect, subMenuRect]) => {
-            expect(menuItemRect.right).to.equal(subMenuRect.left)
+            expect(menuItemRect.right).to.be.approximately(
```

Comment:
Was wondering why this can't be asserted exactly. With the Select tests I'm working on in another PR the assertions with popper placement seem to work with exact comparisons.

Reply:
Also, if my findings in the other PR are correct, these tests will probably have to be updated to at least use `should` so that cypress keeps trying for a while instead of asserting once.

Reply:
I happened to be getting floats here were ever so slightly different. Not quite sure why.

I agree that we _should_ be using `should` 🤣 but shall we address that more holistically when we tackle issues #87 & #86? 
_(bearing in mind that we want to release v5 ASAP so we can freeze the ui-core and ui-widgets repos)_

Or should I just fix this now?

Reply:
Addressing it later seems fine to me 👍